app/controllers/oauths_controller.rb

```diff
@@ -9,14 +9,14 @@ def oauth
   def callback
     provider = auth_params[:provider]
     if @user = login_from(provider)
-      flash[:info] = "Logged in using #{provider.titleize}!"
+      flash[:info] = t('auth.logged_in', provider: provider.titleize)
       redirect_to root_path
     else
       if logged_in?
         link_account(provider)
         redirect_to root_path
       else
-        flash[:alert] = 'You are required to link your GitHub account before you can use this feature. You can do this by clicking "Link your Github account" after you sign in.'
+        flash[:alert] = t('auth.link_after_sign_in')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.